src/schema/v2/partner/BulkOperation/bulkUpdateArtworksMetadataMutation.ts

```diff
@@ -298,6 +306,7 @@ export const bulkUpdateArtworksMetadataMutation = mutationWithClientMutationId<
         provenance: metadata.provenance,
         published: metadata.published,
         signature: metadata.signature,
+        ...transformSignatureFieldsToGravityFields(metadata.signatureTypes),
```

Comment:
[The signatureType enum will be transformed into Gravity's signature fields.
](https://github.com/artsy/metaphysics/blob/484b229c898cb7340046cd3c28e29fc92b47a54e/src/schema/v2/artwork/artworkSignatureTypes.ts#L33)